packages/tkg/bundle/config/packageinstalls/tkr-vsphere-resolver-service-account.yaml

```diff
@@ -0,0 +1,42 @@
+#@ load("@ytt:data", "data")
+
+#@ if data.values.metadata.infraProvider == "vsphere":
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tkr-vsphere-resolver-package-sa
+  namespace: #@ data.values.tkrSourceControllerPackage.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-group: "tkr-vsphere-resolver-packageinstall/serviceaccount-0"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tkr-vsphere-resolver-package-role
+  annotations:
+    kapp.k14s.io/change-group: "tkr-vsphere-resolver-packageinstall/serviceaccount-0"
+rules:
+  #  RBAC rules to create PackageInstall CR and service accounts
+  - apiGroups: ["*"]
```

Comment:
Is this too large permission for the vm template resolver?

Reply:
it is to keep consistent with all other pi rbac within this folder